server/services/cms.js

```diff
@@ -22,6 +23,8 @@ class CmsService {
     switch (type) {
       case 'node--page':
         return this.#cmsApi.get(new BasicPageQuery(location));
+      case 'node--moj_pdf_item':
+        return this.#cmsApi.get(new PdfPageQuery(location));
       case 'node--moj_radio_item':
```

Comment:
this should probably have a test for pdfs like `returns audio content provided by CMS service`